shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/impl/MySQLDALVisitor.java

```diff
@@ -297,6 +299,39 @@ public ASTNode visitSetVariable(final SetVariableContext ctx) {
         return result;
     }
     
+    @Override
+    public ASTNode visitSetName(final SetNameContext ctx) {
+        SetStatement result = new SetStatement();
+        if (null != ctx.characterSetName_() || null != ctx.DEFAULT()) {
+            VariableAssignSegment characterSet = new VariableAssignSegment();
+            VariableSegment variable = new VariableSegment();
+            variable.setVariable("charset");
+            characterSet.setVariable(variable);
+            String assignValue = (null != ctx.DEFAULT()) ? ctx.DEFAULT().getText() : ctx.characterSetName_().getText();
+            characterSet.setAssignValue(assignValue);
+        }
+        if (null != ctx.collationName_()) {
```

Comment:
Do you think it is better to rename `collationName_`  to `collationName`?
`characterSetName_` is the same case.

Reply:
Ok, next pr. @tristaZero